toplevel/coqloop.ml

```diff
@@ -393,3 +393,24 @@ let rec loop ~time ~state =
                   str"Please report" ++
                   strbrk" at " ++ str Coq_config.wwwbugtracker ++ str ".");
       loop ~time ~state
+
+(* Default toplevel loop *)
+let warning s = Flags.(with_option warn Feedback.msg_warning (strbrk s))
+
+let drop_args = ref None
+let loop ~args ~state =
+  drop_args := Some args;
+  let open Coqargs in
+  print_emacs := args.print_emacs;
+  (* We initialize the console only if we run the toploop_run *)
+  let tl_feed = Feedback.add_feeder coqloop_feed in
+  if Dumpglob.dump () then begin
+    Flags.if_verbose warning "Dumpglob cannot be used in interactive mode.";
+    Dumpglob.noglob ()
+  end;
+  let _ = loop ~time:args.time ~state in
+  (* Initialise and launch the Ocaml toplevel *)
+  Coqinit.init_ocaml_path();
+  Mltop.ocaml_toploop();
+  (* We let the feeder in place for users of Drop *)
+  Feedback.del_feeder tl_feed
```

Comment:
I don't get the comment: it says we leave the thing in place, but the line removes it.
(I know it was already there, but I guess the author is you anyway)

Reply:
It means that we delete the feeder _after_ the OCaml toploop exists, instead of _before_. I'll make that clear.